docs/kratos/concepts/credentials/username-email-password.mdx

```diff
@@ -77,9 +77,6 @@ identity:
       # url: base64://...
 ```
 
-If you don't know what that means, please read the [Identity Schema Chapter](../../concepts/identity-schema.mdx) in the docs'
```

Comment:
should we not link the new identity schema doc here?

Reply:
Added this back